stubs/psutil/psutil/_common.pyi

```diff
@@ -184,7 +186,7 @@ class pio(NamedTuple):
     write_bytes: int
 
 class pionice(NamedTuple):
-    ioclass: Any
+    ioclass: int
```

Comment:
Is there a function that returns a `pionice` instance? The class definition is the only thing I found by grepping in typeshed. We should annotate the function too, so that this annotation is actually useful.

Reply:
`Process.ionice()` should return it, but it returns `int` right now. Will fix.